packages/algob/src/lib/compile.ts

```diff
@@ -23,11 +24,11 @@ export class CompileOp {
   // Will throw an exception if the source file doesn't exists.
   // @param filename: name of the TEAL code in `/assets` directory.
   //   (Examples: `mysc.teal, security/rbac.teal`)
-  //   MUST have a .teal extension
+  //   MUST have a .teal or .msig extension
   // @param force: if true it will force recompilation even if the cache is up to date.
   async ensureCompiled (filename: string, force: boolean): Promise<ASCCache> {
-    if (!filename.endsWith(tealExt)) {
-      throw new Error(`filename "${filename}" must end with "${tealExt}"`); // TODO: convert to buildererror
+    if (!filename.endsWith(tealExt) && !filename.endsWith(msigExt)) {
+      throw new Error(`filename "${filename}" must end with "${tealExt}" or "${msigExt}"`); // TODO: convert to buildererror
```

Comment:
you can restore these changes, as this part of code is never used for loading msig

Reply:
I think it will. This is added for getting `msig` from text file (`sample-text-asc.msig`). The TEAL part of `msig` will be compiled here (for a `.msig` file). Or do you mean something else @amityadav0 ?

Reply:
Ok, understood
but we are also loading artifact using `readArtifact(filename);` and that is loading Object as `ASCCache`
that may result in some problem, (not sure will have to test it)

Reply:
Ok, noted. Will check.

Reply:
It is good, checked it